modules/ROOT/pages/api-led-prerequisites.adoc

```diff
@@ -0,0 +1,16 @@
+= Build an API Tutorial: Prerequisites
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:experimental:
+
+Before you begin your API-led journey, verify that you have the basic tools and access needed.
```

Comment:
```suggestion
Before you begin your API-led journey, verify that you have the basic tools and access needed:
```

Reply:
a complete sentence gets a period (unless it uses "the following" phrase)
a sentence fragment gets a colon (or a stce using "the following")